app/test/unit/main-process/spell-checker-menu-test.ts

```diff
@@ -0,0 +1,58 @@
+import { getSpellCheckLanguageMenuItemOptions } from '../../../src/main-process/menu/build-spell-check-menu'
+import { getAvailableSpellcheckerLanguages } from '../../helpers/menus/available-spellchecker-languages-helper'
+
+describe('spell-checker-menu', () => {
+  it('spell checker language is english && user language is english', () => {
+    const spellcheckerLanguage = ['en-US']
+    const userLanguageCode = 'en-US'
+
+    const spellcheckerMenuItem = getSpellCheckLanguageMenuItemOptions(
+      userLanguageCode,
+      spellcheckerLanguage,

```

Comment:
Given `userLanguageCode` and `spellcheckerLanguage` are not used anywhere else, you can just get rid of them:
```suggestion
    const spellcheckerLanguage = ['en-US']
    const userLanguageCode = 'en-US'

    const spellcheckerMenuItem = getSpellCheckLanguageMenuItemOptions(
      'en-US',
      ['en-US'],

```
(You might need to run the linter if you accept these changes).

This also applies to the other tests

Reply:
@sergiou87 initially I implemented the tests without `userLanguageCode` and `spellcheckerLanguage`. I added them after that to increase readability because the parameters looked too similar even though they are different types.